templates/partial/_channel-map.html

```diff
@@ -32,6 +32,10 @@
     </div>
   </div>
 </div>
+<div class="juju-notification">
```

Comment:
I feel like Vanilla typography should handle this spacing instead of the bespoke styling.

Reply:
@anthonydillon the default vanilla spacing didn't have sufficient margins and was inconsistent across viewports

Reply:
Appreciate the replies, thanks.